### ISSUE TITLE
Tool tip drop-down menu title update

### ISSUE THREAD

**Issue by mattwojo** [microsoft/terminal#7105]
The tool tip that pops up when hovering over the drop-down menu carrot in Windows Terminal pops up the same "New tab" tool tip as hovering over the new tab + icon. 

The drop-down menu carrot should have it's own separate tool tip -- something like "Menu" or "Settings & profiles". 

![image](https://user-images.githubusercontent.com/7566797/88719211-c29f7e80-d0d7-11ea-8d59-5fd528316911.png)

### Environment

```none
Windows version 2004, Build 19546.1000
Windows Terminal version: 1.0.1401.0

```

### Steps to reproduce

Hover over menu carrot

### Expected behavior

Tool tip that pops up should be different than + icon tool tip for new tab... should instead read "Menu" or "Settings & profiles"

### Actual behavior

Tool tip reads "New tab" for both new tab icon AND menu icon

**Comment by DHowett** [comment]
This is a /duplicate of #6078, blocked on https://github.com/microsoft/microsoft-ui-xaml/issues/2064 (same root cause) :smile: Thanks!
